src/Symfony/Component/AutoMapper/Extractor/ReflectionExtractor.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AutoMapper\Extractor;
```

Comment:
Shouldn't this class be moved in PropertyInfo too?

Reply:
Yes it should, like the serializer normalizer bridge.